src/main/java/edu/cornell/kfs/module/purap/businessobject/options/IWantNoteComparator.java

```diff
@@ -0,0 +1,16 @@
+package edu.cornell.kfs.module.purap.businessobject.options;
+
+import java.util.Comparator;
+
+import edu.cornell.kfs.module.purap.businessobject.IWantNote;
+
+public class IWantNoteComparator implements Comparator {
+
+    @Override
+    public int compare(final Object o1, final Object o2) {
```

Comment:
Is this needed if the BO IWantNote is updated to implement comparable?

Reply:
Good point. I was basing this on the AccountTypeValuesFinder example.